README.md

```diff
@@ -83,30 +83,48 @@ parameters:
 
 Be aware that it may hide genuine errors in your application.
 
-## Console command analysis
+## Analysis of Symfony Console Commands
 
-You can opt in for more advanced analysis by providing the console application from your own application. This will allow the correct argument and option types to be inferred when accessing `$input->getArgument()` or `$input->getOption()`.
+You can opt in for more advanced analysis of [Symfony Console Commands](https://symfony.com/doc/current/console.html)
+by providing the console application from your own application. This will allow the correct argument and option types to be inferred when accessing `$input->getArgument()` or `$input->getOption()`.
 
-```
+```neon
 parameters:
 	symfony:
 		console_application_loader: tests/console-application.php
 ```
 
-For example, in a Symfony project, `console-application.php` would look something like this:
+Symfony 4:
 
 ```php
+// tests/console-application.php
+
 require __DIR__.'/../config/bootstrap.php';
 $kernel = new \App\Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
 return new \Symfony\Bundle\FrameworkBundle\Console\Application($kernel);
 ```
 
-You may then encounter an error with PhpParser:
+Symfony 5:
+
+```php
+// tests/console-application.php
+
+use App\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Dotenv\Dotenv;
 
-```bash
-Compile Error: Cannot Declare interface PhpParser\NodeVisitor, because the name is already in use
+require __DIR__ . '/../vendor/autoload.php';
+
+(new Dotenv())->bootEnv(__DIR__ . '/../.env');
```

Comment:
Could you unify the two scripts more? Either have `use` statements in everything for all names, or FQN for everything... Thanks!

Reply:
Sure :-)